drivers/platform/stm32/stm32_gpio.c

```diff
@@ -107,6 +107,10 @@ static int32_t _gpio_init(struct gpio_desc *desc,
 #ifdef GPIOH
 	else if (pextra->port == GPIOH)
 		__HAL_RCC_GPIOH_CLK_ENABLE();
+#endif
+#ifdef GPIOJ
+	else if (pextra->port == GPIOJ)
+		__HAL_RCC_GPIOJ_CLK_ENABLE();
```

Comment:
this commit has two independent changes, let's split it into two for better history tracking

Reply:
sorry, can you help me with this. Looks like I have merged the master branch into this branch to even the remote and upstream branches and pushed a new commit related to pwm error handling, so the initial commits about gpio changes has moved down below merge changes from master (logs added below). I am not sure how can I remove the older commit Or do I need to remove gpio changes and push them again as two separate commits?

commit 97d1c372f100b71dd693bcf5b31a37b5fb1f54a7 (HEAD -> stm32-pwm-support, origin/stm32-pwm-support)
Author: mahphalke <Mahesh.Phalke@analog.com>
Date:   Thu Sep 16 13:27:32 2021 +0530

    Fixed the review comments related to pwm error handlings

    Signed-off-by: mahphalke <Mahesh.Phalke@analog.com>

commit afbdb3966090a3b60df2674d8f284fa23544b4b1
Merge: 59ac048a fa40d58a
Author: Mahesh Phalke <62383520+mphalke@users.noreply.github.com>
Date:   Thu Sep 16 13:17:41 2021 +0530

    Merge branch 'analogdevicesinc:master' into stm32-pwm-support

commit fa40d58aa19a2116731c4b568bf47b27f30daca7 (upstream/master, origin/master, origin/HEAD, master)
Author: Janani Sunil <janani.sunil@analog.com>
Date:   Wed Aug 25 14:27:25 2021 +0530

    drivers:temperature:adt7420:Add new supported devices of adt7xx family

    Signed-off-by: Janani Sunil <janani.sunil@analog.com>

commit f392b59a4756b8f4e50b54ae846f20848ed0eec1 (upstream/testbranch)
Author: APorumb <andrei.porumb@analog.com>
Date:   Fri Sep 10 10:23:19 2021 +0300

    iio_adpd1080 build fix using new irq_platform_ops